CMakeLists.txt

```diff
@@ -215,6 +209,31 @@ if(PLASMA_TARGETS STREQUAL "Ethereal")
     add_definitions(-DSTREAM_LOGGER)
 endif(PLASMA_TARGETS STREQUAL "Ethereal")
 
+
+set_property(CACHE PLASMA_PIPELINE PROPERTY STRINGS "DirectX" "OpenGL")
```

Comment:
It looks like what you've actually added is OpenGL ES support -- should we distinguish that here in case a Desktop GL version is added later?  Or were you planning on doing some sort of dynamic detection of the two APIs like Qt 5.5 does?